go.mod

```diff
@@ -7,13 +7,14 @@ require (
 	cosmossdk.io/log v1.3.1
 	cosmossdk.io/math v1.3.0
 	github.com/avast/retry-go/v4 v4.6.0
-	github.com/babylonlabs-io/babylon v0.9.1
+	github.com/babylonlabs-io/babylon v0.9.3-0.20240904123958-b1e255a85e76
```

Comment:
Is this version the latest main or latest released tag?

Reply:
I think the release tag, took this version as suggested by @KonradStaniec in [this PR](https://github.com/babylonlabs-io/btc-staker/pull/31#pullrequestreview-2280339986) to btc staker